oap-server/server-starter/src/main/resources/application.yml

```diff
@@ -543,3 +543,17 @@ receiver-telegraf:
   selector: ${SW_RECEIVER_TELEGRAF:default}
   default:
     activeFiles: ${SW_RECEIVER_TELEGRAF_ACTIVE_FILES:vm}
+
+aws-firehose:
+  selector: ${SW_RECEIVER_AWS_FIREHOSE:default}
+  default:
+    host: ${SW_RECEIVER_AWS_FIREHOSE_HTTP_HOST:0.0.0.0}
+    port: ${SW_RECEIVER_AWS_FIREHOSE_HTTP_PORT:13800}
```

Comment:
Consider using 12801 as the default port number, we can increase by 1 from 11800 for gRPC ports and increase by 1 from 12800 for http ports.